.circleci/config.yml

```diff
@@ -224,11 +236,6 @@ commands:
           name: "Run scripts/run-tox-scenario"
           command: scripts/run-tox-scenario '<< parameters.pattern >>'
       - save_tox_cache
-      - when:
```

Comment:
Note: none of the tox jobs store coverage (profiler), so I'm removing this as we are migrating away from tox.